network/init.sh

```diff
@@ -14,6 +14,10 @@ RPCPORT=${RPCPORT:-26657}
 RESTPORT=${RESTPORT:-1317}
 ROSETTA=${ROSETTA:-8081}
 
+ORACLE_ADDRESS=${ORACLE_ADDRESS:-localhost:8080}
+ORACLE_METRICS_ENABLED=${ORACLE_METRICS_ENABLED:-true}
+ORACLE_CLIENT_TIMEOUT=${ORACLE_CLIENT_TIMEOUT:-2s}
```

Comment:
2s seems too much given 1s local/cloud block time. let's switch to something lesser? like 500ms

Reply:
Is it related to block time?

Reply:
well, this timeout [is used](https://github.com/skip-mev/slinky/blob/6530aaa172d11fd5bc35019dead6fddc5bd86218/abci/ve/vote_extension.go#L136) in ExtendVote in each block, and each validator makes a request to oracle with this timeout. so, if oracle is unavailable, node will be stuck until the timeout is reached. correct me if I'm wrong

Reply:
Nice, that's a good catch actually.

We need to keep this in mind when we will create recommended params of oracle to vals

Reply:
it should be low for sure as far as I understand. a validator would better submit a nil vote extension rather than being stuck waiting for either long timeout or parent context cancellation

Reply:
hm, interesting, i would investigate into this